pkg/middleware/auth.go

```diff
@@ -325,6 +326,16 @@ func NewAuth(authConfig *config.ServiceAuthConfig) *Auth {
 		return &Auth{Verifier: NoAuth{}}
 	}
 
-	return &Auth{Verifier: NewJWTVerifier(authConfig.HMACSecrets)}
+	decodedSecrets := [][]byte{}
+	for _, hmacSecret := range authConfig.HMACSecrets {
+		decodedSecret, err := jwtauth.DecodeConfigValue(hmacSecret)
+		if err != nil {
+			log.Error().Err(err).Msg("failed to decode HMAC secret")
```

Comment:
same comment as before. if we have one bad out of many good ones, do we still disable (return nil)?